edge/pkg/edged/edged_pods.go

```diff
@@ -530,3 +532,331 @@ func (e *edged) removeOrphanedPodStatuses(pods []*v1.Pod) {
 
 	e.statusManager.RemoveOrphanedStatuses(podUIDs)
 }
+
+// GetPodCgroupParent gets pod cgroup parent from container manager.
+func (e *edged) GetPodCgroupParent(pod *v1.Pod) string {
+	/*pcm := e.containerManager.NewPodContainerManager()
+	_, cgroupParent := pcm.GetPodContainerName(pod)
+	return cgroupParent*/
+	return "systemd"
```

Comment:
should we track via an issue that we have harcoded this ? what's the impact ?

Reply:
Ok..will raise issue